### PR TITLE
Web service API requests

### PR DESCRIPTION
**Pull request by nicolasvergoz**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { SplashScreen } from '@ionic-native/splash-screen';
 import { ScreenOrientation } from '@ionic-native/screen-orientation';
 
 import { ListContactsPage } from '../pages/list-contacts/list-contacts';
+import {ApiProvider} from "../providers/api/api";
```
Comment (Nilmanduil) [inline src/app/app.component.ts]: Please fix the TSLint error before merging

File: src/app/app.module.ts [hunk 2]
```diff
@@ -12,6 +12,11 @@ import { ListContactsPage } from '../pages/list-contacts/list-contacts';
 import { DetailsContactPage } from '../pages/details-contact/details-contact';
 import { AddEditContactPage } from '../pages/add-edit-contact/add-edit-contact';
 import { SignUpPage } from '../pages/sign-up/sign-up';
+import { ApiProvider } from '../providers/api/api';
+import { ContactsProvider } from '../providers/contacts/contacts';
+import { AuthProvider } from '../providers/auth/auth';
+import { ErrorProvider } from '../providers/error/error';
+import {HttpClientModule} from "@angular/common/http";
```
Comment (Nilmanduil) [inline src/app/app.module.ts]: Idem

File: src/providers/api/api.ts [hunk 3]
```diff
@@ -0,0 +1,135 @@
+import {HttpClient, HttpHeaders} from '@angular/common/http';
```
Comment (Nilmanduil) [inline src/providers/api/api.ts]: Idem

**Comment by nicolasvergoz** [comment]
Code cleaned !

**Pull request merged**